### ISSUE TITLE
`rbw code UUID` doesn't work with v1.10.0

### ISSUE THREAD

**Issue by yusufaktepe** [doy/rbw#170]
Since the last update `rbw code UUID` stopped working, while `rbw code NAME` still works.

```
$ rbw code dcbaf857-8618-4e4a-ba76-aae000b44a15
rbw code: couldn't find entry for 'dcbaf857-8618-4e4a-ba76-aae000b44a15': no entry found
$ rbw code vault.bitwarden.com
644009
```

**Comment by doy** [comment]
sorry about that - fixed in #169

**Comment by yusufaktepe** [comment]
> sorry about that - fixed in #169

Thank you.

I think `edit` needs a fix as well. I didn't notice it before.
